api/definitions.yaml

```diff
@@ -88,6 +88,8 @@ MessageContent:
       $ref: "#/MessageBodyMarkdown"
     payment_data:
       $ref: "#/PaymentData"
+    due_date:
+      $ref: "#/Timestamp"
```

Comment:
we should require this timestamp to have a date with the timezone

Reply:
done, now must be in the format "2018-10-13T00:00:00.000Z" with UTC tz

Reply:
excellent, pls update the specs so that the docs [here](https://teamdigitale.github.io/digital-citizenship/api/public.html) are clear about this

Reply:
see below